torch/csrc/cuda/Module.cpp

```diff
@@ -1080,6 +1080,11 @@ static void registerCudaPluggableAllocator(PyObject* module) {
     return (storage_impl->data_ptr().get_deleter() == alloc->raw_deleter());
   });
 
+  m.def("_storage_Use_Count", [](size_t storage_impl_ptr) {
+    c10::StorageImpl* storage_impl = (c10::StorageImpl*)storage_impl_ptr;
+    return c10::raw::weak_intrusive_ptr::use_count(storage_impl);
```

Comment:
I feel like you want to only ever return 1 or >1
The reason is that any value >1 should not be relied upon ever due to multithreading.

Reply:
We don't do a similar thing for tensor use count.

Reply:
Personally for debugging, it's helpful to know how many references there are to something, as opposed to simply "> 1".